README.md

```
@@ -86,6 +86,8 @@ Gitcord is a local, offline‑first automation engine that reads GitHub activity
 6. [AOSSIE Discord — Gitcord thread](https://discord.com/channels/1022871757289422898/1465995983791063140)
 4. [Technical Documentation](TECHNICAL_DOCUMENTATION.md) - Architecture and design
 5. [Docker Guide](docs/DOCKER.md) - Docker setup and mentor-friendly deployment
+6. [Handover (easy)](HANDOVER-EASY.txt) - One USB file: `gitcord-handover pack` → `restore`
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 9; Actual: 6; Style: 1/2/3

(MD029, ol-prefix)